modules/release-notes/README.adoc

```diff
@@ -0,0 +1,69 @@
+= Single-sourcing release notes
+
+== Single-sourcing general purpose release notes
+
+The general purpose release notes content is single-sourced from link:https://issues.redhat.com/browse/RHIDP[the JIRA project].
+
+.Prerequisites
+* In link:https://issues.redhat.com/secure/Dashboard.jspa?selectPageId=12364101#SIGwKWmOqDCVBoapBCJiDqhoiKInaroYEg9j2PldSYMUcQVVVdrFHVDUxs1uBtQolXVZgDTdZwjXSCNOA1u11Xeog9Xjb100DUNCh2jwi0TVgzWteeg2FC1TmbT1TUrftGiFHa2CjQtp2TX1DnIGgGCIgEKI+iQfrUlV2AvkJQA[the JIRA project]:
+** *Fix Version/s* is set.
+** *Release Notes Text* is set using following format: a level 0 title followed by descriptive content.
```

Comment:
Please provide an example of what a level 0 title looks like, eg., 

```
= issue description goes here
```

Reply:
```suggestion
** *Release Notes Text* is set using following format: a level 0 title followed by descriptive content, such as:
+ 
.*Release Notes Text* sample
----
= Configurable PVC mounting for containers

Previously, the default Persistent Volume Claim (PVC) could only be mounted to the Backstage container. With this update, you can now configure which container(s) the PVC should be mounted to, providing greater flexibility in storage management. 
----
```